src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,35 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    public static final int TOTAL_MAX_SIZE = 12;
+    public static final int TOTAL_EMPTY = -1;
+    public static final int TOTAL_FULL = 11;
+    private int[] numbers = new int[TOTAL_MAX_SIZE];
 
-    public int total = -1;
+    public int total = TOTAL_EMPTY;
 
     public void countIn(int in) {
         if (!isFull())
             numbers[++total] = in;
     }
 
     public boolean checkCall() {
-        return total == -1;
+        return total == TOTAL_EMPTY;
     }
 
     public boolean isFull() {
-        return total == 11;
+        return total == TOTAL_FULL;
     }
 
     protected int peekaboo() {
         if (checkCall())
-            return -1;
+            return TOTAL_EMPTY;
         return numbers[total];
     }
 
     public int countOut() {
         if (checkCall())
-            return -1;
+            return TOTAL_EMPTY;
```

Comment:
Czy mógłbyś to zastąpić stałą reprezentującą wartość domyślną zwracaną przez wyliczankę, wspólną dla metod countOut i peekaboo?